src/parsing/css-style-value-parsing.js

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//     You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//     See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function(internal) {
+
+  var parsers = {};
+  parsers[CSSNumberValue] = internal.parsing.consumeNumber;
```

Comment:
It'd probably be better to use a type enum or use a proper map (https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Map) here instead. It stringifies the function if you do this:

```javascript
>> var o = {}
>> o[CSSStyleValue] = 5
>> Object.keys(o)
["function CSSStyleValue() {\n  throw new TypeError('CSSStyleValue cannot be instantiated.');\n}"]
```